marklogic-data-hub/src/test/java/com/marklogic/hub/step/impl/WriteStepRunnerTest.java

```diff
@@ -99,7 +99,25 @@ public void testLoadStepRunnerParameters() {
         Assertions.assertEquals("csv", wsr.inputFileType, "Input file type should be 'csv'");
         Assertions.assertEquals("json", wsr.outputFormat, "Output format should be 'json'");
         Assertions.assertEquals(".*/input,''", wsr.outputURIReplacement, "output URI replacement format should be '.*/input,'''");
+        //if 'outputURIReplacement' is set and 'outputURIPrefix' is not set, it will remain null
```

Comment:
Can you put this into a separate test method? The current method name - "testLoadStepRunnerParameters" - is fairly vague as to what it's testing. This change really calls for a new test method that is scoped to outputURIPrefix - the name could just be "testOutputURIPrefix", and it would include these scenarios pertaining to outputURIPrefix.